colossalai/shardformer/layer/qkv_fused_linear.py

```diff
@@ -620,6 +634,154 @@ def forward(self, input_: Tensor) -> Tensor:
             return output, self.bias
 
 
+class GPT2FusedLinearConv1D(ParallelModule):
```

Comment:
You'd better use a new name. This class does not use TP. Why name it "1D"?

Reply:
> You'd better use a new name. This class does not use TP. Why name it "1D"?

Fixed in https://github.com/hpcaitech/ColossalAI/pull/6150/commits/25da23de41cc2d7957c671016f7df15988312a63